superset/connectors/sqla/models.py

```diff
@@ -566,7 +566,7 @@ def get_sqla_query(  # sqla
 
         cols = {col.column_name: col for col in self.columns}
         metrics_dict = {m.metric_name: m for m in self.metrics}
-
+        print(metrics_dict)
```

Comment:
remove print

Reply:
Thanks @hughhhh ! will revise it in another PR. I will close this PR since it will introduce some compatibility issue in master.